bin/main.rb

```diff
@@ -20,7 +21,7 @@
 
 game.players(first_player, second_player)
 
-game.board_game.draw_board
+board.draw_board
```

Comment:
I don't think that you were supposed to do that. `Board` no longer exist outside of a `Game`

Reply:
This line is responsible to show the board in the command line before the game begin.